content/en/apps/guides/debugging/obtaining-logs.md

```diff
@@ -22,6 +22,6 @@ The easiest way is to use `medic-logs`, a tool that comes with [`medic-conf`](ht
 medic-logs anInstance gardener
 ```
 
-See [medic-logs documentation](https://github.com/medic/medic-conf#medic-logs) for more details.
+See [medic-logs documentation](https://github.com/medic/medic-conf#user-content-fetching-logs) for more details.
```

Comment:
this looks right but just takes me to the top of the medic-conf page
https://github.com/medic/medic-conf#medic-logs

Reply:
Should probably be this link: https://github.com/medic/medic-conf#fetching-logs